core/src/ibc/channel_04/mod.rs

```diff
@@ -14,10 +14,10 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+mod log;
 mod manager;
 pub mod types;
 
-
```

Comment:
Could you create another commit for this change?

Reply:
done